src/Bridges/bridgeoptimizer.jl

```diff
@@ -222,37 +222,72 @@ function MOI.set(b::AbstractBridgeOptimizer,
     return MOI.set(b.model, attr, indices, values)
 end
 
+# function MOI.get(::MOI.ModelLike, attr::MOI.AbstractConstraintAttribute,
+#     bridge::AbstractBridge)
+# throw(ArgumentError("Constraint bridge of type `$(typeof(bridge))` does not support accessing the attribute `$attr`."))
+# end
+
 # Constraint attributes
 function MOI.get(b::AbstractBridgeOptimizer,
                  attr::MOI.AbstractConstraintAttribute,
                  ci::CI)
     if is_bridged(b, typeof(ci))
-        if MOI.is_set_by_optimize(attr)
-            MOI.get(b, attr, bridge(b, ci))
-        else
+        if attr isa MOI.ConstraintName || attr isa MOI.ConstraintFunction ||
+                attr isa MOI.ConstraintSet
             MOI.get(b.bridged, attr, ci)
+        else
+            MOI.get(b, attr, bridge(b, ci))
         end
     else
         MOI.get(b.model, attr, ci)
     end
 end
-## Setting names
-function MOI.supports(b::AbstractBridgeOptimizer, attr::MOI.ConstraintName,
-                      Index::Type{<:CI})
-    if is_bridged(b, Index)
-        return MOI.supports(b.bridged, attr, Index)
+function MOI.supports(b::AbstractBridgeOptimizer,
+                      attr::MOI.AbstractConstraintAttribute,
+                      IndexType::Type{CI{F, S}}) where {F,S}
+    if is_bridged(b, IndexType)
+        if attr isa MOI.ConstraintName || attr isa MOI.ConstraintFunction ||
```

Comment:
`ConstraintFunction` and `ConstraintSet` are not copyable (see `is_copyable`) so `supports(model, MOI.ConstraintFunction(), ...)` is not defined. We should not worry about these two attributes here

Reply:
I removed `ConstraintFunction` and `ConstraintSet` from the condition.